draft-ietf-httpbis-digest-headers.md

```diff
@@ -579,8 +579,14 @@ When Integrity fields are used in a trailer section, the field-values are receiv
 Eager processing of content before the trailer section prevents digest validation, possibly leading to
 processing of invalid data.
 
-Not every hashing algorithm is suitable for use in the trailer section, some may require to preprocess
-the whole content before sending a message (e.g., see {{?I-D.thomson-http-mice}}).
+One of the benefits of using Integrity fields in a trailer section is that it
+allows hashing of bytes as they are sent or received. However, it is possible to
```

Comment:
I don't think you want "or received" here. If the digest is in a header, you can still hash as you're receiving the content.